data/net/minecraft/world/entity/SlotAccess.mapping

```diff
@@ -1,3 +1,21 @@
 CLASS net/minecraft/world/entity/SlotAccess
+	METHOD forContainer (Lnet/minecraft/world/Container;I)Lnet/minecraft/world/entity/SlotAccess;
+		ARG 0 inventory
+		ARG 1 slot
+	METHOD forContainer (Lnet/minecraft/world/Container;ILjava/util/function/Predicate;)Lnet/minecraft/world/entity/SlotAccess;
+		ARG 0 inventory
+		ARG 1 slot
+		ARG 2 accessHandler
+	METHOD forEquipmentSlot (Lnet/minecraft/world/entity/LivingEntity;Lnet/minecraft/world/entity/EquipmentSlot;)Lnet/minecraft/world/entity/SlotAccess;
+		ARG 0 entity
+		ARG 1 slot
+	METHOD forEquipmentSlot (Lnet/minecraft/world/entity/LivingEntity;Lnet/minecraft/world/entity/EquipmentSlot;Ljava/util/function/Predicate;)Lnet/minecraft/world/entity/SlotAccess;
+		ARG 0 entity
+		ARG 1 slot
+		ARG 2 accessHandler
```

Comment:
**RE:** Perhaps instead of `accessHandler`, this could be named `stackFilter` as its used to filter which items can be set in that slot.
```suggestion
		ARG 2 stackFilter
```